### ISSUE TITLE
No colour on linux tty terminal

### ISSUE THREAD

**Issue by THe-SP00K** [charmbracelet/lipgloss#31]
On Ubuntu - everything works fine in the Gnome Terminal and $TERM is **xterm-256color**

If I switch to tty console where the $TERM is **linux** then I get no colour. I expected it fall back to 4-bit, which is supported

**Comment by THe-SP00K** [comment]
Actually - the issue looks to be upstream
https://github.com/muesli/termenv/issues/37

**Comment by strickolas** [comment]
Hey,

I've been fighting with this problem for the past two days. I don't know a ton about rendering in the terminal, but I thought I'd share what I've learned so far, and maybe you can confirm or correct my observations.

When I run a bubbletea program colored using lipgloss on my gnome shell, where `$TERM` is *xterm-256color* everything looks great. I then ran the same binary on  a Fedora 34 server, and Ubuntu 18.04 server (both whose $TERM is set to *linux* by default), and both times yielded a pretty horrible experience. I rendered to bubbletea program using the altscreen option, and when running my TUI while there is still some output on the terminal, the output gets mangled.

I tried setting `$TERM` to *xterm-256color, but to no avail. It turns out, color profiles are located in `/usr/share/terminfo/`. I scp'd the color profile from my workstation located at `/usr/share/terminfo/x/xterm-256color`, ran `export TERM=xterm-256color` on the servers, and ... still not working correctly.  I did some experimentation with different color profiles, and I've found  that `export TERM=xterm-color` does the job (at least enough for things to render correctly-ish). This doesn't change the fact that rendering an altscreen tui while the current terminal session has a lot of text on it will cause the tui to get mangled a bit, but this can be solved by running `clear` before the terminal.

I'm not entirely satisfied by this, and I was wondering if you -- or anyone else in the community -- has a solution or workaround to get pretty tuis on terminals where $TERM = linux

**Comment by muesli** [comment]
Setting `COLORTERM` to `truecolor` will enable 24bit color rendering support. Setting `TERM` to `*256color` will enable extended ANSI colors (8-bit), while setting it to `*color` will enable regular ANSI color support. We're not currently using `terminfo` for that.

Are you running `screen` or similar terminal multiplexers?

**Comment by strickolas** [comment]
Wow, you reply fast. See, this is where my limited knowledge of terminals comes in. I'm not entirely sure which multiplexer I'm using, or even how to check that.

**Comment by muesli** [comment]
If you don't know `screen`, you're probably not using it :smiley:

Can you double-check that `export TERM=term-256color` isn't working? It definitely should!

**Comment by strickolas** [comment]
```
$ export TERM=term-256color
$ reset
reset: unknown terminal type term-256color
Terminal type?
```

`export TERM=*256color` also yields similar results.

**Comment by muesli** [comment]
Sorry, that was a typo: I meant `xterm-256color`.

**Comment by strickolas** [comment]
Ah, `xterm-256color` works -- but only because I scp'd over the color profile. It also doesn't work the same as it does on my gnome shell on Fedora34 workstation. Greens are teal, and pastel reds/pinks are grey.

**Comment by muesli** [comment]
This will be fixed in the next `termenv` release. As soon as we bump its version in `lipgloss`, ANSI-16 should automatically be supported on the Linux console.

**Comment by muesli** [comment]
This should be fixed in lipgloss v0.5.0.
